tutorials/colors/colormapnorms.py

```diff
@@ -69,6 +70,46 @@
 fig.colorbar(pcm, ax=ax[1], extend='max')
 plt.show()
 
+###############################################################################
```

Comment:
This should definitely be moved under TwoSlopeNorm, please, as it is a specialization of that norm.